testing/btest/core/analyzer-confirmation-violation-info.zeek

```diff
@@ -1,4 +1,5 @@
 # @TEST-DOC: The SSL analyzer picks up on the traffic in pppoe-over-qing, but then raises analyzer_violation_info
+# @TEST-REQUIRES: ! grep -q "#define ENABLE_SPICY_SSL" $BUILD/zeek-config.h
```

Comment:
Assuming this is due to missing parity: could you add a comment/TODO explaining what is missing to not require this?